### ISSUE TITLE
MenuItem: AppMenuItem.vue npm reference issue in file in Ultima and Apollo and probabably other templates too.

### ISSUE THREAD

**Issue by nexgen-ui** [primefaces/primevue#6467]
### Describe the bug

AppMenuItem.vue has a reference to @primeuix/utils, which is not imported in package.json.

import { calculateScrollbarWidth, getViewport } from '@primeuix/utils/dom';

It would be better to add a reference to @primevue/core which exposes those references anyway.
Also would need to add a reference to it in package.json

import { calculateScrollbarWidth, getViewport } from '@primevue/core/utils';

### Reproducer

Not required

### PrimeVue version

4.00

### Vue version

4.x

### Language

ALL

### Build / Runtime

Vite

### Browser(s)

Chrome 128

### Steps to reproduce the behavior

1. Add typescript support to the project and compile it.
2. The reference breaks because it doesn't exist in package.json

### Expected behavior

All package references should be defined in package.json

**Comment by tugcekucukoglu** [comment]
PrimeVue uses `@primeuix/utils` internally, but you can add it package.json yourself.

Thanks for your attention.

**Comment by nexgen-ui** [comment]
I know I can fix it myself. I would like the template to be fixed so that if someone downloads it, they don't encounter the same issues. This is the third bug that I have logged that has just been closed without an actual fix. Also what I am saying is that @primevue/core is probably the better package to add when doing template development because it exports the packages from the internal package you mentioned.
